### DISCUSSION TITLE
Piggy bank API broken

### DISCUSSION THREAD

**Discussion by 4e868df3** [firefly-iii/firefly-iii#7561] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

The date format isn't consistent when querying the piggy bank API. It appears to be returning an inconsistent date format.

### Debug information

Endpoint = `/api/v1/piggy-banks`

Fields affected = `['start_date', 'target_date']`

### Expected behaviour

Previous, expected format: `%Y-%m-%dT%H:%M:%S%z`

Current, unexpected, possibly inconsistent, format: `%Y-%m-%d`

### Steps to reproduce

1. Query the piggy bank endpoint
2. observe that the date format has changed

### Additional info

Firefly v6.0.11 returns the incorrect format.

**Comment by JC5** [comment]
That was changed deliberately, and the API version has been increased because of it. The API docs reflect this as well.

**Reply by 4e868df3** [reply]
🤦‍♂️ sorry totally my bad. Thanks for pointing that out, I'll make it up to you!

**Reply by JC5** [reply]
Haha it's not a problem!
